Jenkinsfile

```diff
@@ -74,6 +78,16 @@ try {
                     '''
             }
         }
+        stage('Build Docker Image') {
```

Comment:
Is there a way we can avoid re-building the Docker image every single Pipeline run? I'm assuming that this nginx container will change *very* infrequently, while the jenkins.io content is rebuilt every few minutes

Reply:
No there isn't, either we rebuild this image every pipeline run, either we move this docker image to another Github repository.
As this image is not specific to www.jenkins.io, we may consider to move it into another project.

Reply:
Fair enough, let's move it into another repository then